docs/docs/target/kubernetes.md

```diff
@@ -101,13 +91,79 @@ Total: 3 (UNKNOWN: 0, LOW: 1, MEDIUM: 0, HIGH: 2, CRITICAL: 0)
 └────────────────┴────────────────┴──────────┴────────┴───────────────────┴──────────────────────────────────┴───────────────────────────────────────────────────┘
 ```
 
+### Taints and Tolerations
+
+The node-collector scan-job will run on every node. In case the node has been tainted, it is possible to add toleration to the scan job for it to be scheduled on the tainted node
+
+- `--tolerations  key1=value1:NoExecute,key2=value2:NoSchedule` this flag wil enable node-collector to be schedule on tainted Node
+
+Example:
+
+```sh
+trivy k8s --report summary --tolerations  key1=value1:NoExecute,key2=value2:NoSchedule
+```
+
+### Non-intrusive
+
+You can control whether the node scan-job (`node-collector`) will run in the cluster. To disable it, add the `--non-intrusive` flag  
+
+- `--non-intrusive` This flag will exclude findings related to Node (infra assessment) misconfigurations
+
+By default, the node scan-job (`node-collector`) will run in the cluster.
+
+Example:
+
+```sh
+trivy k8s --report summary --non-intrusive
+```
+
+### Skip-images
+
+You can control whether Trivy will scan and download the cluster resource images. To disable this feature, add the --skip-images flag.
+
+- `--skip-images` flag will prevent the downloading and scanning of images (including vulnerabilities and secrets) in the cluster resources.
+
+By default, all cluster resource images will be downloaded and scanned.
+
+Example:
+
+```sh
+trivy k8s --report summary --skip-images
+```
+
+### Include/Exclude Kinds
+
+You can control which kinds of resources will be discovered using the `--include-kinds` or `--exclude-kinds` comma-separated flags:
```

Comment:
I have never heard the word `kinds` in relation to Kubernetes, this is a bit strange to me -- could you share your reasoning for the naming?

Reply:
well yes each K8s resource is of a particular kind but I have not seen the plural used -- I have also never seen it outside of a reference for particular K8s types -- thus, my confusion, to me it sounds strange

Reply:
I think it's fairly common in k8s lingo (also in plural). for example, search for "kind" in these community docs:

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md
https://kyverno.io/docs/writing-policies/match-exclude/
https://open-policy-agent.github.io/gatekeeper/website/docs/constrainttemplates
https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/#parameter-resources